extensions/handlebars/language-configuration.json

```diff
@@ -22,5 +22,24 @@
 		{ "open": "\"", "close": "\"" },
 		{ "open": "<", "close": ">" },
 		{ "open": "{", "close": "}" }
-	]
+	],
+	"wordPattern": {
+		"pattern": "(-?\\d*\\.\\d\\w*)|([^\\`\\~\\!\\@\\$\\^\\&\\*\\(\\)\\=\\+\\[\\{\\]\\}\\\\\\|\\;\\:\\'\\\"\\,\\.\\<\\>\\/\\s]+)",
+		"flags": "g"
```

Comment:
I believe the 'g' is mandatory for all word patterns. I has to do with the way we use the word pattern regexes.
So I would remove it from the language-configurations and set it where we create the RegExp object.

So the extension can do `"wordPattern": "(-?\\d..." *` (unless `i` or `u` is needed as well)